### PR TITLE
Project partner logos

### PR DESCRIPTION
**Pull request by Elscrux**

### CODE REVIEW

**Comment by Elscrux** [comment]
@schweikart Can we merge this?

**Comment by schweikart** [comment]
The suggested layout works but I think we can improve the layout a bit. I think we should re-organize the legal links (imprint + privacy) and we should maybe make the project partner logos less prominent.

This is the current layout:

Screenshot from 2023-04-07 10 54 44

Here is my take on the design (pushed to branch `project-partner-logos-alternate-layout`):

Screenshot from 2023-04-07 11 51 32

Also, the 4flow logo seems to be have a different tone of white for its background. Do you think we can get a transparent version of their logo?

Screenshot from 2023-04-07 11 55 34

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import type { NextPage } from "next";
+import { URL } from "next/dist/compiled/@edge-runtime/primitives/url";
```
Comment (schweikart) [inline src/pages/index.tsx]: unnecessary import

File: public/4flow.svg
Comment (schweikart) [inline public/4flow.svg:1]: as mentioned in my comment: can we get a transparent version of this logo?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -42,22 +42,49 @@ const Home: NextPage = () => {
...
-          justifyContent={"space-between"}
+        <SimpleGrid
+            columns={3}
+            borderTopWidth={1}
+            width={"100%"}
+            paddingTop={"1rem"}
+            textAlign={"left"}
+            justifyContent={"space-between"}
         >
           <VStack>
```
Comment (schweikart) [inline src/pages/index.tsx]: the `VStack` wrapper can be omitted for these single-element nodes

**Review by schweikart — CHANGES_REQUESTED** [review]

**Comment by Elscrux** [comment]
Seeing as you've done most of the work fixing these issues I think we can switch to that branch instead. I uploaded the transparent 4flow logo so now hopefully everything is in place to be merged.

**Comment by Elscrux** [comment]
I've merged the alternate style branch and included the last fixes

**Review by schweikart — APPROVED** [review]

**Comment by schweikart** [comment]
perfect, thanks :raised_hands:

**Pull request merged**